components/LessonTabs/LessonTabs.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import NavCard from '../NavCard'
+
+type Lesson = {
+  title: string
+  docUrl?: string | null
+  slug: string
+}
+
+const LessonTabs = ({
+  lesson,
+  activeTab
+}: {
+  lesson: Lesson
+  activeTab: string
+}) => {
+  const tabs = [
+    ...(lesson.docUrl ? [{ text: 'lesson', url: lesson.docUrl }] : []),
+    { text: 'challenges', url: `/curriculum/${lesson.slug}` },
+    { text: 'exercises', url: `/exercises/${lesson.slug}` },
+    {
+      text: 'mentor exercises',
+      url: `/curriculum/${lesson.slug}/mentor`
```

Comment:
If the paths are static, can we use the constants [here]?(https://github.com/garageScript/c0d3-app/blob/master/constants/index.ts)

Reply:
Yeah, great idea!